src/scheduler/SnapshotServer.cpp

```diff
@@ -1,40 +1,39 @@
-#include <faabric/scheduler/MpiWorldRegistry.h>
```

Comment:
You can check a cleaner diff [here](https://github.com/faasm/faabric/blob/zeromq/src/scheduler/SnapshotServer.cpp).